src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -93,9 +96,22 @@ protected override bool Visit(ITestSkipped testSkipped)
 
         protected override bool Visit(ITestStarting testStarting)
         {
+            if (showProgress)
```

Comment:
I believe you need to lock on consoleLock to prevent this output from being intertwined with other messages or when the console colors are changed for other messages.